tests/scripts/deploy_admission_controller.sh

```diff
@@ -81,39 +81,4 @@ EOF
 
 echo "Successfully deployed cert-manager"
 
-echo "Creating Issuer and Certificate"
-cat <<EOF | kubectl create -f -
-apiVersion: cert-manager.io/v1
-kind: Issuer
-metadata:
-  name: selfsigned-issuer
-  namespace: ${NAMESPACE}
-spec:
-  selfSigned: {}
----
-apiVersion: cert-manager.io/v1
-kind: Certificate
-metadata:
-  name: rook-admission-controller-cert
-  namespace: ${NAMESPACE}
-spec:
-  dnsNames:
-  - ${SERVICE_NAME}
-  - ${SERVICE_NAME}.${NAMESPACE}.svc
-  - ${SERVICE_NAME}.${NAMESPACE}.svc.cluster.local
-  issuerRef:
-    kind: Issuer
-    name: selfsigned-issuer
-  secretName: rook-ceph-admission-controller
-EOF
-
-echo "Successfully created Issuer and Certificate"
-
-echo "Deploying webhook config"
-< "${BASE_DIR}"/webhook-config.yaml \
```

Comment:
Why removing this, I think it's still useful so users know how to deploy the webhook.

Reply:
added back